server/start.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+set -e
+npm run build-config
+exec node server.js
```

Comment:
This bash file is used for starting service on EC2 instances, so we need to source a config file, ensure node version through nvm (just like previous start.sh did).

Just keep the previous file
```suggestion
#!/bin/bash -eu

. "${NVM_DIR}/nvm.sh"
. version.sh

nvm ls "${START_NODE_VERSION}" > /dev/null || { echo "ERROR: Node version ${START_NODE_VERSION} not installed"; exit 1; }
nvm use --delete-prefix "${START_NODE_VERSION}"

. config/env.sh

npm run build-config
exec node server
```